src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -82,9 +91,14 @@ def handle_exception(e):
 
         @app.before_request
         def log_before_request_info():
+            app.config["last_request_time"] = datetime.now()
             g.start = time.perf_counter()
-            app.logger.debug("Headers: %s", request.headers)
-            app.logger.debug("Body: %s", request.get_data())
+            app.logger.debug(
+                "Last request time: %s, Headers: %s, Body: %s",
+                app.config["last_request_time"],
+                request.headers,
+                request.get_data(),
```

Comment:
I remember we had the discussion that wheter log data will not expose secret? but how this work for connection related api?

Reply:
removed connection request body